src/components/manage/Blocks/Maps/MapsData.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
I don't think MapsData.jsx is needed, maybe just put this stuff in the MapsSidebar directly? In principle, we never do separate components for the sidebar in our blocks, just stash it all in the Edit component, but here it should be ok, as the edit has more stuff in it any way.

Reply:
I removed it. thank you